Stabilization changes for `xsd-downloader`

leebradley opened the pull request.

CLEckhardt commented:
Needs to add `RUN mkdir /var/cache/grid` to the daemon Dockerfile, otherwise tested and LGTM

leebradley commented:
> Needs to add `RUN mkdir /var/cache/grid` to the daemon Dockerfile, otherwise tested and LGTM

Added to the Dockerfile. I've also added a volume mount to the docker-compose.yaml files for the cache directory to persist it through rebuilds.

CLEckhardt reviewed (approved):
tested
